echopype/tests/convert/test_parsed_to_zarr.py

```diff
@@ -158,3 +161,5 @@ def test_direct_to_zarr_integration(path_model: str, raw_file: str,
             ed_zarr, ed_no_zarr = compare_zarr_vars(ed_zarr, ed_no_zarr, var_to_comp, grp)
 
         assert ed_zarr[grp].identical(ed_no_zarr[grp])
+
+    del ed_zarr
```

Comment:
Wondering here too about checking for successful deletion of the temp dir, as in the above comment.

Reply:
That sounds like a good idea, I will add that.